admin-panel/src/main/resources/application.properties

```
@@ -27,3 +27,7 @@ spring.security.oauth2.client.registration.cryptic-oauth.provider=cryptic-oauth
 spring.security.oauth2.client.provider.cryptic-oauth.issuer-uri=${OIDC_ISSUER_URL:}
 #spring.security.oauth2.client.provider.cryptic-oauth.user-name-attribute=${OIDC_USERNAME_ATTRIBUTE:}
 server.forward-headers-strategy=native
+
+#gameserver
+gameserver.apiToken = ${API_TOKEN:}
+gameserver.serverUrl = ${SERVER_URL:}
```

```suggestion
cryptic.admin-panel.api-token=${API_TOKEN:}
cryptic.admin-panel.server-url=${SERVER_URL:}
```

Properties files without spaces.